### PR TITLE
Fix grammer in indices/refresh.asciidoc

### PR DESCRIPTION
**Pull request by mattayes**
# Changes

* Fix grammer issue in indices/refresh.asciidoc.

### CODE REVIEW

**Review by jrodewig — APPROVED** [review]
LGTM. Thanks for fixing this @mattayes.

**Comment by jrodewig** [comment]
@elasticmachine test this please

File: docs/reference/indices/refresh.asciidoc [hunk 1]
```diff
@@ -42,7 +42,7 @@ using the <<index-refresh-interval-setting,`index.refresh_interval`>> setting.
 
 [IMPORTANT]
 ====
-Refreshes are a resource-intensive.
+Refreshes are a resource-intensive operation.
```
Comment (jrodewig) [inline docs/reference/indices/refresh.asciidoc]:
```suggestion
Refreshes are resource-intensive.
```

Comment (jrodewig) [inline docs/reference/indices/refresh.asciidoc]: Actually, can we just remove the `a` here.
```suggestion
Refreshes are resource-intensive.
```

**Comment by jrodewig** [comment]
@mattayes I made a small suggestion that should fix it. After rereading, I believe your fix has a subject/verb agreement issue.

**Review by jrodewig — CHANGES_REQUESTED** [review]
@mattayes I made a small suggestion that should fix it. After rereading, I believe your fix has a subject/verb agreement issue.

**Comment by mattayes** [comment]
@jrodewig Change made.

**Review by jrodewig — APPROVED** [review]
Thanks @mattayes!

**Comment by jrodewig** [comment]
@elasticmachine test this please

**Comment by jrodewig** [comment]
### Backport commits 
master: 5d289e3
7.x: 344c218
7.5: c3fa878

**Comment by mattayes** [comment]
Thanks for the quick review/merge @jrodewig!

**Pull request merged**